src/organizations/components/DeleteOrgReasonsForm.tsx

```diff
@@ -31,17 +17,25 @@ enum VariableItems {
   OTHER_REASON = 'Other reason',
 }
 
-function DeleteOrgReasonsForm({errors = {}}: ComponentProps) {
-  const [workingVariable, setWorkingVariable] = useState('')
+function DeleteOrgReasonsForm() {
+  const [workingVariable, setWorkingVariable] = useState('USE_CASE_DIFFERENT')
   const {
     shortSuggestion,
     isShortSuggestionEnabled,
     suggestions,
     setShortSuggestionFlag,
     setShortSuggestion,
     setSuggestions,
+    reason,
+    setReason,
   } = useContext(DeleteOrgContext)
 
+  useEffect(() => {
+    if (reason !== VariableItems[workingVariable]) {
+      setReason(VariableItems[workingVariable])
+    }
+  }, [reason, workingVariable])
```

Comment:
why is this in a useEffect and not in function that's invoked?

Reply:
Initially, this is not set. So, we need this to have a default value stored in the context. Unless we store these `VariableItems` in the context and provide a default one?

Reply:
What do you mean, you’re providing an initial value in the state above right?

Reply:
Like, the setting of state should be triggered based on a user interaction of selecting a property, not based on the byproduct of hook, unless I’m missing something